app/assets/stylesheets/active_admin/components/_tables.scss

```diff
@@ -33,7 +33,7 @@ table.index_table {
     }
 
     &.sortable a {
-      background: image-url('active_admin/orderable.png') no-repeat 0 4px; padding-left: 13px;
+      background: url("#{$imagePath}/orderable.png") no-repeat 0 4px; padding-left: 13px;
```

Comment:
How does this work with precompilation in Rails with sprockets? Are you sure that the paths get updated? My main concern is that sprockets is looking for that `image-url` method so I don't think this will work as expected.

Reply:
Hum that's something I completely oversaw indeed. 
I'll try to run precompilation on my test app and keep you updated. In case it doesn't work, do you have any idea how to handle that? I'm thinking of adding a `sed` or something like that when copying files to `src/`, or have different "flavours" of those 2 files

Reply:
I'm not actually sure how to handle that. It seems sassc-rails just registers a sass function called `asset-url` (the `image-url` is just an alias) so we'd need something here for that. My first thought is to check the webpacker docs and any example Rails apps using webpacker for CSS as well. I'd start there and see what you find out. Thanks!

Reply:
What I'd love to do though is just replace the images with inline SVG using data URLs. These are simple icons so it would work and then we'd be able to use the SCSS in both without issue as we would no longer be dependent on external assets. Another nice bonus is users wouldn't be precompiling any assets here other than the CSS and JS files.

Reply:
Using data uri is a brilliant idea! Since those icons were really, really low-def converting them was pretty much impossible, so we should find completely new icons in SVG, and the ones I found were way bigger than current files (eg. https://www.iconfinder.com/icons/211614/arrow_b_down_icon). 
I used current png files converted in base64, and used them as data uris to keep it simple for now. I that okay for you?